test/it/tenant_it.js

```diff
@@ -9,6 +9,8 @@ var Tenant = require('../../lib/resource/Tenant');
 describe('Tenant', function() {
   var client, tenant;
 
+  this.timeout(120000);
```

Comment:
Really minor thing. But for readability I'd rather write this as `60 * 2 * 1000`.